src/framework/mocha/MochaRuntime.m

```diff
@@ -1656,15 +1657,24 @@ static JSValueRef MOFunction_callAsFunction(JSContextRef ctx, JSObjectRef functi
 
     // Perform the invocation
     @try {
-        value = MOFunctionInvoke(function, ctx, argumentCount, arguments, exception);
+        value = MOFunctionInvoke(function, ctx, argumentCount, arguments, &error);
     }
     @catch (NSException *e) {
         debug(@"caught exception whilst invoking function %@: %@", function, e);
-
-        // Catch ObjC exceptions and propogate them up as JS exceptions
-        if (exception != nil) {
-            *exception = [[Mocha runtimeWithContext:ctx] JSValueForObject:e];
-        }
+        
+        error = [[Mocha runtimeWithContext:ctx] JSValueForObject:e];
+    }
+    
+    // Catch ObjC exceptions and propogate them up as JS exceptions
+    if (error != NULL) {
+        JSStringRef errorMessage = JSStringCreateWithUTF8CString("An Obj-C exception occured.");
```

Comment:
occured -> occurred

Reply:
Funnily enough, there are 23 instances of `occured` in the codebase 😄